### PR TITLE
migrate to yarn from npm #102

### PR DESCRIPTION
**Pull request by Gri-ffin**

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -50,12 +50,13 @@ jobs:
...
-        # Run npm scripts #
+        # Run yarn scripts #
         ###################
 
-      - run: npm ci
-      # - run: npm run build --if-present
-      - run: npm test --if-present
+      - run: yarn install --frozen-lockfile
+      - run: yarn run build
+      # - run: yarn run build --if-present
```
Comment (meganindya) [inline .github/workflows/CI.yml:58]: remove this

File: README.md [hunk 2]
```diff
@@ -248,19 +248,19 @@ Windows) this repository using
 6. The _Linux Debian 10.7_ (_buster_) _shell_ in the _docker container_ named _musicblocks_ is
 spawned and standard input/output is connected to the terminal.
 
-    _**Node**_ (_Node.js Runtime_), _**npm**_ (_Node Package Manager_), _**tsc**_ (_TypeScript
+    _**Node**_ (_Node.js Runtime_), _**yarn**_ (_Node Package Manager_), _**tsc**_ (_TypeScript
```
Comment (meganindya) [inline README.md:251]: *Yarn Package Manager

**Review by meganindya — CHANGES_REQUESTED** [review]
Please see the comments. Other than that it's better to classify README changes as `docs` commits and project setup changes as `chore`. So, your first commit is a `chore` while the last two are `docs`. In fact, you can squash down your last two commits into one. An interactive rebase will get these done.

File: .github/workflows/CI.yml (hunk 1 above)
Comment (Gri-ffin) [reply .github/workflows/CI.yml:58]: do you mean the comment?

Comment (meganindya) [reply .github/workflows/CI.yml:58]: yes

**Review by meganindya — APPROVED** [review]

**Comment by meganindya** [comment]
Can you fix the merge conflicts and rebase your branch? I want to run the workflows before merging PRs (even though this PR shouldn't really have linting issues). There was something broken with the Super Linter which has been fixed now.

**Pull request closed without merging**